### PR TITLE
tModLoader

### PR DESCRIPTION
**Pull request by Zoey2936**
I know there is already a pr to add a template for this (#107), but I created this additional pr to fit my other pr of terraria server (#121 and #127)

### CODE REVIEW

**Comment by Zoey2936** [comment]
Also, it does not need unzip to be installed

File: terraria-tmodloader/terraria-tmodloader.json [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+  "install": [
+    {
+      "files": [
+        "https://github.com/tModLoader/tModLoader/releases/download/v${tversion}/tModLoader.Linux.v${tversion}.tar.gz"
+      ],
+      "type": "download"
+    },
+    {
+      "commands": [
+        "mv ${rootDir}/${mp}/enabled.json ${rootDir}",
```
Comment (LordRalex) [inline terraria-tmodloader/terraria-tmodloader.json]: RootDir is pretty much deprecated, as it's usage is very limited. Your commands should not need to use it, even though it works with it. Can you please remove them?

**Review by LordRalex — CHANGES_REQUESTED** [review]

File: terraria-tmodloader/terraria-tmodloader.json (hunk 1 above)
Comment (Zoey2936) [reply terraria-tmodloader/terraria-tmodloader.json]: I can remove them from the installation and the starting command, but when I remove them or replace them with ./ inside the configuration file, the server do not load the settings of the world's safe folder and file

**Comment by Zoey2936** [comment]
without the other ${rootDir} it will not work
edit: it does work

File: terraria-tmodloader/terraria-tmodloader.json (hunk 1 above)
Comment (Zoey2936) [reply terraria-tmodloader/terraria-tmodloader.json]: I found why it is not working without ${rootDir}, it is now removed

Comment (Zoey2936) [reply terraria-tmodloader/terraria-tmodloader.json]: I fixed it in all three templates

**Pull request merged**